api.planx.uk/send/bops.ts

```diff
@@ -58,80 +54,63 @@ const sendToBOPS = async (req: Request, res: Response, next: NextFunction) => {
 
   const bopsFullPayload = await $admin.generateBOPSPayload(payload?.sessionId);
 
-  useProxy({
+  const bopsResponse = await axios({
+    method: "POST",
+    url: target,
     headers: {
-      ...(req.headers as Record<string, string | string[]>),
+      "Content-Type": "application/json",
       Authorization: `Bearer ${process.env.BOPS_API_TOKEN}`,
     },
-    pathRewrite: () => "",
-    target,
-    selfHandleResponse: true,
-    onProxyReq: (proxyReq, req) => {
-      req.body = bopsFullPayload;
-      fixRequestBody(proxyReq, req);
-    },
-    onProxyRes: responseInterceptor(
-      async (responseBuffer, proxyRes, req, res) => {
-        // Mark session as submitted so that reminder and expiry emails are not triggered
-        markSessionAsSubmitted(payload?.sessionId);
-
-        const bopsRawResponse = responseBuffer.toString("utf8");
-
-        let bopsResponse: { id: string } | undefined;
-        try {
-          bopsResponse = JSON.parse(bopsRawResponse);
-        } catch (e) {
-          res.statusCode = 502; // Bad Gateway - invalid response from the upstream server
-          return bopsRawResponse;
-        }
-
-        const applicationId = await adminClient.request(
-          gql`
-            mutation CreateBopsApplication(
-              $bops_id: String = ""
-              $destination_url: String = ""
-              $request: jsonb = ""
-              $req_headers: jsonb = ""
-              $response: jsonb = ""
-              $response_headers: jsonb = ""
-              $session_id: String = ""
-            ) {
-              insert_bops_applications_one(
-                object: {
-                  bops_id: $bops_id
-                  destination_url: $destination_url
-                  request: $request
-                  req_headers: $req_headers
-                  response: $response
-                  response_headers: $response_headers
-                  session_id: $session_id
-                }
-              ) {
-                id
-                bops_id
-              }
+    data: bopsFullPayload,
+  }).then(async (res: AxiosResponse<{ id: string }>) => {
```

Comment:
Would a `.catch()` block following this be a sensible precaution and help us debug?